agoraStreaming/src/Navigation/styles.ts

```diff
@@ -0,0 +1,31 @@
+import {StyleSheet, ViewStyle} from 'react-native';
+
+type HeaderStyleType = {
+  container: ViewStyle;
+  title: Text;
+  wrapperSectionIcons: ViewStyle;
+  wrapperIcon: ViewStyle;
+};
+export const styles = StyleSheet.create<HeaderStyleType>({
+  container: {
+    flexDirection: 'row',
+    paddingVertical: 10,
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  title: {
+    color: 'white',
+  },
+  wrapperSectionIcons: {
+    flexDirection: 'row',
+  },
+  wrapperIcon: {
```

Comment:
I saw it above in the code. Lets try to prevent duplicate the lines
![image](https://user-images.githubusercontent.com/45641770/142228499-60e55af6-67ca-4e9d-96d3-fea249ceab37.png)